<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>串口配置</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打开设备</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>32</y>
       <width>60</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>串口选择</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>系统参数</string>
    </property>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>32 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128 Bytes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256 Bytes</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>70</y>
       <width>60</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>安全等级</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>数据包大小</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>79</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>36</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>45</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>191</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>指纹库</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>181</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>模块操作</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>录入指纹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>连续录入指纹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>匹配指纹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>对比选中的指纹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>删除选中的指纹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>清空指纹库</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>系统提示</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>460</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>接收数据</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>硬件信息</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>传感器名称</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>产品型号</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>当前设备波特率</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>数据包大小</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>设备地址</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>软件版本号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>指纹库容量</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>传感器类型</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>厂家名称</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>安全等级</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
